context/app/static/js/shared-styles/icons/index.js

```diff
@@ -2,6 +2,11 @@ import styled from 'styled-components';
 import BubbleChartIcon from '@material-ui/icons/BubbleChart';
 import PersonIcon from '@material-ui/icons/Person';
 import TableChartIcon from '@material-ui/icons/TableChart';
+import AccountBalanceIcon from '@material-ui/icons/AccountBalance';
```

Comment:
Can you make sure all the icons are the rounded icons? We decided to use the rounded icon library from Materials. So for example,
`import AccountBalanceIcon from '@material-ui/icons/AccountBalanceRounded'`

Reply:
Will do! (Also, if there's just a small typo in a PR, click the "±" in the toolbar to suggest edits: The author can accept the changes from gitup without going back to the editor.)